multichain-testing/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "packageManager": "yarn@4.5.3",
   "devDependencies": {
-    "@agoric/cosmic-proto": "0.4.1-dev-08f8549.0",
+    "@agoric/cosmic-proto": "dev",
```

Comment:
I suppose `latest` is not an option (aka the upgrade-18 packages are not up to date enough?)

Reply:
`0.5.0-u18` may work but I think it's better to integrate with head